tests/sentry/api/test_event_search.py

```diff
@@ -2100,6 +2102,38 @@ def test_implicit_and_explicit_tags(self):
             [["ifNull", ["tags[project_id]", "''"]], "=", "123"]
         ]
 
+    def test_in_syntax(self):
+        project_2 = self.create_project()
+        group = self.create_group(project=self.project, short_id=self.project.next_short_id())
+        group_2 = self.create_group(project=project_2, short_id=self.project.next_short_id())
+        assert (
+            get_filter(
+                f"project.name:[{self.project.slug}, {project_2.slug}]",
+                params={"project_id": [self.project.id, project_2.id]},
+            ).conditions
+            == [["project_id", "IN", [project_2.id, self.project.id]]]
+        )
+        assert (
+            get_filter(
+                f"issue:[{group.qualified_short_id}, {group_2.qualified_short_id}]",
+                params={"organization_id": self.project.organization_id},
+            ).conditions
+            == [["issue.id", "IN", [group.id, group_2.id]]]
+        )
+        assert (
+            get_filter(
+                f"issue:[{group.qualified_short_id}, unknown]",
+                params={"organization_id": self.project.organization_id},
+            ).conditions
+            == [[["coalesce", ["issue.id", 0]], "IN", [0, group.id]]]
+        )
+        assert get_filter("environment:[prod, dev]").conditions == [
+            [["environment", "IN", {"prod", "dev"}]]
```

Comment:
Why does this produce a set?

Reply:
It has just been implemented this way for a long time:
https://github.com/getsentry/sentry/blob/6e80a3cc2f7796aa8c034dac5b226128926d4974/src/sentry/api/event_search.py#L893
Preexisting test:
https://github.com/getsentry/sentry/blob/6e80a3cc2f7796aa8c034dac5b226128926d4974/tests/sentry/api/test_event_search.py#L2387

Reply:
Weird. OK well I guess if it ain't broke don't fix it.

Reply:
It might be worth standardising it, I don't see any real reason that it's a set. But going to avoid extra changes in this pr